src/studio/nuxeo-studio/working-in-studio/branding.md

```diff
@@ -391,168 +392,93 @@ history:
+-   A picture covering all the screen. This is the strategy chosen by Nuxeo.
+-   A logo with a transparent background with a background color.
 
+{{#> callout type='warning' }} While selecting your background, be careful not to use heavy files to ensure a quick display to your users.
 {{/callout}}
 
-### Declaring a New CSS Class and Its Rules
-
-You can add style classes in your widgets in [forms]({{page page='form-layouts'}}), [content views]({{page page='content-views'}}), and [tab layout]({{page page='tabs'}}) to manage the rendering of your elements.
-
-Let's add a background color on a widget's container:
-
-<div class="table-scroll"><table class="hover"><tbody><tr><th colspan="1">
-
-1.  Widget Editor
-
-</th><th colspan="1">2\. CSS Editor</th><th colspan="1">3\. Result</th></tr><tr><td colspan="1">
-
-![]({{file name='blue_background_config.png'}} ?w=300,border=true,thumbnail=true)
+### Selecting a Background Image  
 
-</td><td colspan="1">
+1.  In `Background Image`, click on **Select Resources**.
+1.  Define options for the image, such as defining the position (in pixel or position) of the image or if the background cover all the screen (with CSS3 / recent browsers).
+1.  Pick a color that will be displayed if the background image is too small and not covering all the screen.
 
-![]({{file name='-blue_background_style.png'}} ?w=300,border=true,thumbnail=true)
+|  Before | Studio Configuration | After |
+|---|---|---|
+| ![]({{file name='branding_picture_nuxeo.png'}} ?w=300,border=true) | ![]({{file name='login_customization.png'}} ?w=300,border=true)  |![]({{file name='branding_picture.png'}} ?w=300,border=true)  |
 
-</td><td colspan="1">![]({{file name='blue_background_container.png'}} ?w=300,border=true,thumbnail=true)</td></tr><tr><td colspan="1">
+### Selecting a Background Video
 
-Define a class in the widget editor.
+1.  Select a background video.
+1.  In `Background Video URLs / MIME Type`, click **Add**.
+1.  Enter the URL and MIME Type of your video.
+1.  Define the options for the video, such as looping or muting the video.
 
-</td><td colspan="1">Declare your blue background in the CSS Editor.
+## Customizing the Login Box
 
-</td><td colspan="1">Enjoy your blue container.</td></tr></tbody></table></div>
+### Selecting a Logo
 
-### Overriding an Existing Rule
+By default, the logo used is the one defined as your Application Logo. You can select another logo to appear in your login box.
 
-You want to change an existing style to fit your needs? In addition to the [Nuxeo UI Style Guide](http://showcase.nuxeo.com/nuxeo/styleGuide/) that explains the default set of CSS existing rules, you will need to use a browser's web inspector to understand what to override.
+1. In the `Background` section, check **No** to Apply Application Logo.
+1. In the `Login logo` section, click **Select Resources** and upload the logo image.
+1. Optional: Define width and height size of the logo.
```

Comment:
`Optional: Define the width and height of the logo.`

Reply:
Done

Reply:
Thank you for the review!